lib/cadet/assessments/answer_types/mcq_answer.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Cadet.Assessments.AnswerTypes.MCQAnswer do
+  @moduledoc """
+  The Assessments.QuestionTypes.MCQQuestion entity represents an MCQ Answer.
+  It comprises of one of the MCQ choices.
+  """
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Cadet.Assessments.QuestionTypes.MCQChoice
+
+  embedded_schema do
+    embeds_one(:answer_choice, MCQChoice)
```

Comment:
Actually now that I think about it, it might be better to give some kind of pseudo-id to our `MCQChoice` per `MCQQuestion`, so that this only needs to store the `MCQChoice`'s id.
So, e.g. for `MCQQuestion` with id 1, it has 4 choices, each with id's 1, 2, 3, and 4. Same thing for `MCQQuestion` with id 2, it has 4 choices, each with id's 1, 2, 3, and 4, and so on and so forth.
This will also make it easier for the API -- the client needs only submit the question id and choice id to submit an answer.
What do you think?

Reply:
Seems to be fine. The only downside I think is that we cannot trace question from an answer but such a case probably won't occur. I think we can go ahead with it.

Reply:
Restructured but for standard 4 options. Should this be variable ?